certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -3985,418 +4992,1389 @@
+                                        "description": "Is Preferred Shipping."
+                                    },
+                                    "isPreferredBilling": {
+                                        "type": "boolean",
+                                        "description": "Is Preferred Billing."
+                                    },
+                                    "isBadAddress": {
+                                        "type": "boolean",
+                                        "description": "Is Bad Address."
+                                    }
+                                }
+                            },
+                            "description": "Addresses."
+                        }
+                    }
                 },
-                "description": {
-                  "type": "string",
-                  "description": "Description.",
-                  "title": "Description",
-                  "x-ms-visibility": "internal",
-                  "default": "Added by Impexium Connector"
-                }
-              },
-              "required": [
-                "callBackUrl",
-                "description",
-                "secret",
-                "typeName",
-                "filters"
-              ]
-            }
-          }
-        ]
-      }
-    },
-    "/api/v1/Webhooks/RequestUpdated": {
-      "x-ms-notification-content": {
-        "description": "When customer request is updated.",
-        "schema": {
-          "$ref": "#/definitions/CustomerRequestPayload"
-        }
-      },
-      "post": {
-        "tags": ["Webhook", "Customer"],
-        "responses": {
-          "200": {
-            "description": "Success",
-            "schema": {}
-          }
-        },
-        "summary": "When customer request is updated",
-        "description": "Triggers when a customer request is added or closed.",
-        "operationId": "When-Request-Updated",
-        "x-ms-trigger": "single",
-        "parameters": [
-          {
-            "name": "Content-Type",
-            "in": "header",
-            "required": true,
-            "type": "string",
-            "default": "application/json",
-            "x-ms-visibility": "internal"
-          },
-          {
-            "name": "body",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "typeName": {
-                  "type": "string",
-                  "description": "Type Name.",
-                  "title": "Type Name",
-                  "x-ms-visibility": "internal",
-                  "default": "Customer.RequestUpdated"
+                "description": "Event Locations."
+            },
+            "categories": {
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "$id": {
```

Comment:
Please remove this property as it's not part of the response.
Check http://abc.mpxdemo.com/api/help/ResourceModel?modelName=CategoryData